Cython/Utility/MemoryView.pyx

```diff
@@ -94,8 +94,11 @@ cdef extern from "<stdlib.h>":
     void free(void *) nogil
     void *memcpy(void *dest, void *src, size_t n) nogil
 
-
-
+# cache the errors used so access to them doesn't need the GIL
+# with Options.cache_builtins
+cdef __pyx_memoryview_IndexError = IndexError
```

Comment:
Why not just declare PyExc_IndexError and friends as external object, and use those?

Reply:
extern `object` isn't allowed but I think this way makes sense so I've declared them as `PyObject*` and added suitable casts inside `_err` and `_err_dim`

Reply:
Did you push that change?

Reply:
No. It looks like I'd stashed it and then forgotten about it. Done now.